Gems/AtomLyIntegration/CommonFeatures/Code/Source/Mesh/MeshComponentController.cpp

```diff
@@ -39,11 +39,12 @@ namespace AZ
                     ->Version(1)
```

Comment:
Update the version number since serializing is changing. We'll also need a version converter to detect cases where the LodOverride is has been set to a real value (not 255) so the correct LodType can be set. Converting is kind of a pain, but in this case it should just come down to using FindElement() to find the LodOverride, check to see if its value is not 255, and in that case call AddElementWithData() to set the new LodType field to SpecificLod. You can search the codebase for "VersionConverter" to find several examples.